addons/sale/models/sale.py

```diff
@@ -740,6 +751,7 @@ def write(self, values):
     company_id = fields.Many2one(related='order_id.company_id', string='Company', store=True, readonly=True)
     order_partner_id = fields.Many2one(related='order_id.partner_id', store=True, string='Customer')
     analytic_tag_ids = fields.Many2many('account.analytic.tag', string='Analytic Tags')
+    is_downpayment = fields.Boolean(string="Is a downpayment")
```

Comment:
I would add an help to explain a bit how this is computed and what are consequences of it.